src/language/mod.rs

```diff
@@ -163,8 +163,23 @@ impl Language {
 
 	/// Get the index of the word in the word list.
 	#[inline]
+	#[allow(unreachable_patterns)]
```

Comment:
Perhaps we should explicitly list the features and not use `allow`?
```rust
// All other languages' wordlists are not lexicographically sorted, so we have to
// resort to linear search
#[cfg(any(feature = "chinese-simplified", feature = "chinese-traditional", feature = "czech", feature = "french", feature = "japanese", feature = "spanish"))]
_ => self.word_list().iter().position(|w| *w == word).map(|i| i as u16),
```

Reply:
I guess it's better to name them explicitly.. Not sure we run our tests on each language individually explciitly.